rules-threat-hunting/windows/process_creation/proc_creation_win_potential_hex_staging_attack.yml

```diff
@@ -0,0 +1,29 @@
+title: Potential Hex Staging Attack
+id: 647c1b89-d717-4a0a-816b-2f129500f76f
+status: experimental
+description: |
+    Detects potential hex staging attack. Threat Actor often delivers malicious executable payload in hexadecimal chunks by writing the data to a temporary file using cmd.exe.
+    The /p parameter takes the hexadecimal payload as input. The encoded payload is later decoded using inbuilt tools e.g certutil for further execution.
+    During investigation, examine the temporary files created, analyze the hex content for malicious patterns, check for subsequent process executions (particularly certutil), and investigate the source of the command execution and the parent process.
+references:
+    - https://unit42.paloaltonetworks.com/espionage-campaign-targets-south-asian-entities
+author: Joseph Kamau
+date: 2025-10-02
+tags:
+    - attack.defense-evasion
+    - attack.t1027.013
+    - detection.threat-hunting
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        Image|endswith: '\cmd.exe'
+        CommandLine|contains|all:
+            - 'set'
+            - '/p='
+           
```

Comment:
This is not indicative of a hex attempt. Without the redirection, this will match with the prompt functionality.

Also the example you linked in the comment https://github.com/SigmaHQ/sigma/pull/5674#discussion_r2436587645 does not fill the file with the text provided in the `/p` command.

Can you provide some evidence on this?